test/run_tests.py

```diff
@@ -121,7 +121,7 @@ def http_server():
 
 def execute_tests(interpreter, browser, rf_options, sauce_username, sauce_key):
     options = []
-    runner = interpreter.split() + ['-m', 'robot.run']
+    runner = interpreter.split() + ['-m', 'robot']
```

Comment:
This change breaks test execution with RF < 3.0. Need to change back to `robot.run` even though that causes an annoying warning with Python 3.6.